packages/design-system/src/styles/components/_Drawer.scss

```diff
@@ -72,6 +72,9 @@
   overflow: auto;
   overscroll-behavior: contain;
 
+  // TODO: Investigate why this is needed.
+  // Block DOM els already have a column layout?
+  // Our supported browsers support flex, so this query not needed.
```

Comment:
Huh, it does seem sus. I traced it back to [this commit](https://github.com/CMSgov/design-system/commit/e19b8ec66508ed19fe82dc8ddcad7cc9325cce2f), and according to that message, these `@supports` queries were added to support IE11. Let's remove them!